content/news/XXXX-XX-XX-github-issues-migration.md

```diff
@@ -0,0 +1,85 @@
+title: Migrated to Github issues
+authors: Nikolaus Einhauser
+tags: infrastructure, development, contribute
+comments: yes
+status: draft
+
+
+Yes everybody, we're officially moving our bug tracking away from Launchpad to
+GitHub. If you've never tried to report a bug in Mixxx or wanted to contribute
+otherwise, it's very likely you have no idea what this post is about, but if you
+do it's even better. No matter if you do or don't, keep reading.
+
+The Mixxx team has traditionally been using Canonical's (the makers of Ubuntu)
+Launchpad platform for hosting their code, managing bug reports and releases.
+Over the years, Launchpad has become increasingly dated, lack of modern
+features, old UI, and bad user experience is the primary reason why many people
+(myself included), stay away from Launchpad. Especially for new contributors the
+mandatory Ubuntu One account was another reason for not to immediately report a
+bug when they encountered it.
+
+We've started transitioning away from Launchpad quite a while ago:  
+Back in 2013 we have transitioned from the Bazaar version control system to Git
+and moved our `main` branch to GitHub. At that point in time, GitHub issues where
+not yet suitable for managing our bugfix and release workflow, thus bug reports
+continued to be handled on Launchpad. This has progressively changed over the
+past couple of years, now GitHub has a "duplicate issue" detection feature, can
+organize bugs in projects and the labeling feature is flexible enough to fillful
+most of our needs.
+
+Going forward, Launchpad will only be used for publishing to our Ubuntu PPA. So
```

Comment:
```suggestion
Going forward, Launchpad will only be using for publishing to our Ubuntu PPA. Hence, 
```

Reply:
the `used` -> `using` change doesn't make sense imo.

Reply:
whoops, sorry for the noise.